DELIVERY.README.txt

```diff
@@ -133,12 +131,8 @@ Contains the final VCF files and indexes, in total 6 files.
 Genomic VCF (gVCF) (and index file) containing sequencing information for both
 variant and non-variant positions. Can be used for downstream cohort calling.
 
---SampleN.clean.dedup.recal.bam.recalibrated.indel.annotated.vcf.gz
---SampleN.clean.dedup.recal.bam.recalibrated.indel.annotated.vcf.gz.tbi
-VCF file containing indel variants called from the recalibrated BAM file and
+--SampleN.clean.dedup.recal.bam.raw.annotated.vcf.gz
+--SampleN.clean.dedup.recal.bam.raw.annotated.vcf.gz.tbi
+VCF file containing variants called from the recalibrated BAM file and
```

Comment:
Dont know what was intentional, only looking at what the pipeline gives me :-). Still we have SNP and INDEL vcf files but these are not annotated. And since we do not recalibrate variants no more I guess creating separate SNP and INDEL vcfs is not necessary.